workspaces/src/types/mod.rs

```diff
@@ -144,7 +144,7 @@ impl PublicKey {
             },
         )?))
     }
-
+    #[allow(dead_code)]
```

Comment:
Let’s be more explicit and hide it behind the interop_sdk feature flag instead of allowing dead code